static/app/views/performance/traceDetails/content.tsx

```diff
@@ -94,14 +99,70 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceNotFound() {
-    const {meta} = this.props;
+    const {meta, traceEventView, traceSlug, organization, location} = this.props;
 
     const transactions = meta?.transactions ?? 0;
     const errors = meta?.errors ?? 0;
 
     if (transactions === 0 && errors > 0) {
+      const errorsEventView = traceEventView.withColumns([
+        {kind: 'field', field: 'project'},
+        {kind: 'field', field: 'title'},
+        {kind: 'field', field: 'issue.id'},
+      ]);
+      errorsEventView.query = `trace:${traceSlug} !event.type:transaction `;
+
       return (
-        <LoadingError message={t('The trace you are looking contains only errors.')} />
+        <DiscoverQuery
+          eventView={errorsEventView}
+          orgSlug={organization.slug}
+          location={location}
+          referrer="api.trace-view.errors-view"
+        >
+          {({isLoading, tableData, error}) => {
+            if (isLoading) {
+              return <LoadingIndicator />;
+            }
+
+            if (error) {
+              return (
+                <Alert type="error" icon={<IconInfo size="md" />}>
+                  <ErrorLabel>
+                    {tct(
+                      'The trace cannot be shown when all events are errors. An error occurred when attempting to fetch these error events: [error]',
+                      {error}
+                    )}
+                  </ErrorLabel>
+                </Alert>
+              );
+            }
+
+            return (
+              <Alert type="error" icon={<IconInfo size="md" />}>
+                <ErrorLabel>
+                  {t('The trace cannot be shown when all events are errors.')}
+                </ErrorLabel>
+
+                <List symbol="bullet" data-test-id="trace-view-errors-list">
+                  {tableData.data.map(data => (
+                    <ListItem key={data.id}>
+                      {tct('[link]: [title]', {
+                        link: (
+                          <Link
+                            to={`/organizations/${organization.slug}/issues/${data['issue.id']}/events/${data.id}`}
+                          >
+                            {data.project}
+                          </Link>
+                        ),
+                        title: data.title,
+                      })}
```

Comment:
Visually, this looks a little strange as we rarely show just the project name without the project icon. I wonder if it makes more sense to show the level instead of the project name like warning, fatal, etc. here.